apps/backend/src/modules/contributors/contributor.repository.ts

```diff
@@ -15,35 +15,52 @@ class ContributorRepository implements Repository {
 
 		const contributor = await this.contributorModel
 			.query()
-			.insert({
-				name,
-			})
+			.insert({ name })
 			.execute();
 
-		return ContributorEntity.initialize(contributor);
+		return ContributorEntity.initialize({ ...contributor, projects: [] });
 	}
 
 	public delete(): ReturnType<Repository["delete"]> {
 		return Promise.resolve(true);
 	}
 
 	public async find(id: number): Promise<ContributorEntity | null> {
-		const item = await this.contributorModel.query().findById(id).execute();
+		const contributor = await this.contributorModel
+			.query()
+			.findById(id)
+			.withGraphFetched("gitEmails");
```

Comment:
I think you can modify `relationMappings` method of `ContributorModel` and use additionally `.withGraphFetched("projects")` here and in other places, so you don't need to "inject" projects using `ProjectService`'s `getProjectsByContributorId`

Reply:
I tried to do it, also discussed this with Liza, and seems like the relation is too complicated, and we cannot describe it in the model.

Reply:
Oh, got it